pkg_resources/_vendor/vendored.txt

```diff
@@ -9,5 +9,7 @@ jaraco.text==3.7.0
 importlib_resources==5.10.2
 # required for importlib_resources on older Pythons
 zipp==3.7.0
+# required for jaraco.functools
+more_itertools==10.2.0
```

Comment:
I added this here because it was listed in `pkg_resources.extern.names`.
The version corresponds to the version that already exists in the `pkg_resources/_vendor` the directory... But that is different from the version in `setuptools/_vendor/vendored.txt`. Maybe we should synchronise these versions so that it makes the life easier for downstream packagers "de-vendoring" setuptools?